src/ucp/core/ucp_worker.c

```diff
@@ -1498,8 +1497,8 @@ void ucp_worker_wait_mem(ucp_worker_h worker, void *address)
     UCS_V_UNUSED ucs_status_t status;
 
     UCP_CONTEXT_CHECK_FEATURE_FLAGS(worker->context, UCP_FEATURE_WAKEUP,
-                                    status, /*no action*/);
-   ucs_arch_wait_mem(address);
+                                    /*no action*/);
+    ucs_arch_wait_mem(address);
```

Comment:
i don't think this feature check is relevant here, @shamisp what do you think?

Reply:
It will work w/o the feature because the only existing implementation that i've found is '/* NOP*/ ', but the API is described `@ingroup UCP_WAKEUP`

Reply:
@yosefe agree. I would remove it for now.

Reply:
removed